package/kernel/linux/modules/netfilter.mk

```diff
@@ -335,6 +335,7 @@ define KernelPackage/ipt-ipset
 	CONFIG_IP_SET_HASH_IPPORT \
 	CONFIG_IP_SET_HASH_IPPORTIP \
 	CONFIG_IP_SET_HASH_IPPORTNET \
+	CONFIG_IP_SET_HASH_IPMAC \
```

Comment:
ipset hash(ip,mac) is already supported; see https://github.com/openwrt/openwrt/blob/master/package/kernel/linux/modules/netfilter.mk#L333